docs/pages/developer-guide.md

```diff
@@ -926,8 +928,8 @@ pods:
         cpus: 1.0
         memory: 256
 ```
-
-Currently an empty YAML dictionary is passed as the body for each network definition under `networks`, since we only support joining virtual networks by name, but in the future it will be possible to specify port mappings and other information in a network definition.
+**Overlay networks**
+The SDK supports having pods join the `dcos` overlay network. For an in-depth explanation of how virtual networks work on DC/OS see the [documentation](https://docs.mesosphere.com/1.9/networking/virtual-networks/#virtual-network-service-dns). When a pod joins an overlay network it gets its own IP address and has access to its own array of ports. Therefore when a pod specifies that it is joining `dcos` we ignore the `ports` resource requirements, because the pod will not consume the ports on the host machine. The DNS for pods on the overlay network is `<task_name>.<framework_name>.autoip.dcos.thisdcos.directory`. Note that this DNS will also work for pods on the host network. Because the `ports` resources are not used when a pod is on the overlay network, we do not allow a pod to be moved from the `dcos` overlay to the host network or vice-versa. This is to prevent potential starvation of the task when the host with the reserved resources for the task does not have the available ports required to launch the task. 
```

Comment:
Suggest:

The SDK enables pods to join the `dcos` overlay network. For an in-depth explanation of virtual networks in DC/OS, see the [documentation].

When a pod joins an over lay network, it gets its own IP address and has access to its own array of ports. Therefore, when a pod specifies that it is joining the `dcos` overlay network, the SDK ignores the `ports` resource requirements because the pod will not consume the ports on the host machine.

The DNS for pods on the overlay network is `<task-name>.<framework-name>.autoip.dcos.thisdcos.directory`. 
**Note:** This DNS will also work for pods on the host network.

Because the `ports` resources are not used when a pod is on the overlay network, the SDK does not allow a pod to be moved from the `dcos` overlay to the host network or vice-versa. This prevents potential starvation of a task when with host with the reserved resources for the task does not have the available ports required to launch the task.